todo_app/view_models/test_todo_list_view_model.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from todo_app.view_models.todo_list_view_model import TodoListViewModel
+from todo_app.data.item import Item
+
+test_items = [
+    Item(1, 'some-name','To Do'),
+    Item(2, 'some-name','Done'),
+    Item(3, 'some-name','To Do'),
+    Item(4, 'some-name','Done'),
+]
+
+def test_view_model_sets_items():
```

Comment:
Very minor but one to consider; there's a small risk here that somebody might modify this `test_items` list during a test, which could impact later tests if run in sequence. For that reason, we might potentially consider using a fixture that generates a fresh set of items per test:
```python
@pytest.fixture
def test_items():
    return [
        Item(1, 'some-name','To Do'),
        Item(2, 'some-name','Done'),
        Item(3, 'some-name','To Do'),
        Item(4, 'some-name','Done'),
    ]

def test_view_model_sets_items(test_items):
```